client/src/pages/CreateProduct.js

```diff
@@ -38,43 +48,43 @@ const useStyles = makeStyles((theme) => ({
     marginTop: theme.spacing(8),
   },
   rootSnackbar: {
-    width: '100%',
-    '& > * + *': {
+    width: "100%",
+    "& > * + *": {
       marginTop: theme.spacing(2),
     },
   },
 }));
 
 const typeOfProduct = [
   {
-    value: 'Cake',
-    label: 'Cake',
+    value: "Cake",
+    label: "Cake",
   },
   {
-    value: 'Cupcake',
-    label: 'Cupcake',
+    value: "Cupcake",
+    label: "Cupcake",
   },
   {
-    value: 'Macarons',
-    label: 'Macarons',
+    value: "Macarons",
+    label: "Macarons",
```

Comment:
looks like your linter config may be different from @austinkwok1997 you can create a prettier config file to stay consistent or sync to make sure your personal dev env is the same